modules/integration/src/test/java/io/ballerina/messaging/broker/integration/standalone/SuiteInitializer.java

```diff
@@ -100,8 +101,18 @@ public void beforeSuite(String port, String sslPort, String hostname, String adm
         }
 
         BrokerAuthConfiguration brokerAuthConfiguration = new BrokerAuthConfiguration();
+        BrokerAuthConfiguration.AuthenticationConfiguration authenticationConfiguration =
+                new BrokerAuthConfiguration.AuthenticationConfiguration();
+        BrokerAuthConfiguration.AuthenticatorConfiguration authenticatorConfiguration =
+                new BrokerAuthConfiguration.AuthenticatorConfiguration();
+        HashMap<String, Object> properties = new HashMap<>();
+        properties.put(BrokerAuthConstants.CONFIG_PROPERTY_JAAS_LOGIN_MODULE,
+                       UserStoreLoginModule.class.getCanonicalName());
+        authenticatorConfiguration.setClassName(JaasAuthenticator.class.getCanonicalName());
+        authenticatorConfiguration.setProperties(properties);
+        authenticationConfiguration.setAuthenticator(authenticatorConfiguration);
+        brokerAuthConfiguration.setAuthentication(authenticationConfiguration);
```

Comment:
Do we need to do all these since we are using the defaults? We should be getting the defaults when we just initialize the config class. WDYT?

Reply:
We use default authenticator as defaults.